### DISCUSSION TITLE
Global tuning

### DISCUSSION THREAD

**Discussion by lenlowe** [bangcorrupt/hacktribe#217] [Ideas]
Hi there!

I recently got back into the Electribe 2 due to Hacktribe and soon found myself looking for global tuning adjustment (440hz+-). Did I just miss it so far? If not: Would it be possible to include this in Hacktribe?

Thanks and best regards,
Len

**Comment by bangcorrupt** [comment]
It's not an option on the e2, but the KingKorg has global tuning along with user definable temperament.  Pitch is mapped to frequency in the CPU firmware before being sent to the DSP, so it might not be too hard to implement some day.

**Reply by lenlowe** [reply]
That sounds great, hope it'll make it into hacktribe! :) Much appreciated
